Description: don't hide compile/link flags so buildd can see them
 buildd wants to check the actual compile/link flags so it can check whether
 the hardening flags have been properly set.  The olsrd build system hides all
 commands.  This patch removes all the hiding by removing the leading '@' from
 the relevant commands.  This has not been forwarded upstream because the
 upstream devs have expressed the desire to keep the build system hiding the
 flags.
 .
 This patch does not change the lib/pud/Makefile since the 'pud' plugin is not
 currently built as part of this package.
 .

Author: Hans-Christoph Steiner <hans@eds.org>
Forwarded: no
Last-Update: <2012-10-02>

diff --git a/Makefile b/Makefile
index 5efdd4a..a9d7fb6 100644
--- a/Makefile
+++ b/Makefile
@@ -64,16 +64,16 @@ default_target: $(EXENAME)
 
 $(EXENAME):	$(OBJS) src/builddata.o
 		@echo "[LD] $@"
-		@$(CC) $(LDFLAGS) -o $@ $^ $(LIBS)
+		$(CC) $(LDFLAGS) -o $@ $^ $(LIBS)
 
 cfgparser:	$(CFGDEPS) src/builddata.o
 		$(MAKE) -C $(CFGDIR)
 
 switch:		
-	@$(MAKECMD) -C $(SWITCHDIR)
+	$(MAKECMD) -C $(SWITCHDIR)
 
 src/builddata.c:
-	@$(RM) "$@"
+	$(RM) "$@"
 	@echo "#include \"defs.h\"" >> "$@" 
 	@echo "const char olsrd_version[] = \"olsr.org -  $(VERS)-git_`git log -1 --pretty=%h`-hash_`./make/hash_source.sh`\";"  >> "$@"
 	@date +"const char build_date[] = \"%Y-%m-%d %H:%M:%S\";" >> "$@" 
@@ -99,8 +99,8 @@ uberclean:	clean clean_libs clean_gui
 	-rm -f $(TAGFILE)
 #	BSD-xargs has no "--no-run-if-empty" aka "-r"
 	find . \( -name '*.[od]' -o -name '*~' \) -not -path "*/.hg*" -type f -print0 | xargs -0 rm -f
-	@$(MAKECMD) -C $(SWITCHDIR) clean
-	@$(MAKECMD) -C $(CFGDIR) clean
+	$(MAKECMD) -C $(SWITCHDIR) clean
+	$(MAKECMD) -C $(CFGDIR) clean
 
 install: install_olsrd
 
@@ -152,7 +152,7 @@ tags:
 		$(TAGCMD) -o $(TAGFILE) $(TAG_SRCS)
 
 rpm:
-		@$(RM) olsrd-current.tar.bz2
+		$(RM) olsrd-current.tar.bz2
 		@echo "Creating olsrd-current.tar.bz2 ..."
 		@./list-excludes.sh | tar  --exclude-from=- --exclude="olsrd-current.tar.bz2" -C .. -cjf olsrd-current.tar.bz2 olsrd-current
 		@echo "Building RPMs..."
@@ -195,100 +195,100 @@ libs_uninstall uninstall_libs:
 #
 
 arprefresh:
-		@$(MAKECMD) -C lib/arprefresh
+		$(MAKECMD) -C lib/arprefresh
 
 arprefresh_clean:
-		@$(MAKECMD) -C lib/arprefresh DESTDIR=$(DESTDIR) clean
+		$(MAKECMD) -C lib/arprefresh DESTDIR=$(DESTDIR) clean
 
 arprefresh_install:
-		@$(MAKECMD) -C lib/arprefresh DESTDIR=$(DESTDIR) install
+		$(MAKECMD) -C lib/arprefresh DESTDIR=$(DESTDIR) install
 
 arprefresh_uninstall:
-		@$(MAKECMD) -C lib/arprefresh DESTDIR=$(DESTDIR) uninstall
+		$(MAKECMD) -C lib/arprefresh DESTDIR=$(DESTDIR) uninstall
 
 bmf:
-		@$(MAKECMD) -C lib/bmf
+		$(MAKECMD) -C lib/bmf
 
 bmf_clean:
-		@$(MAKECMD) -C lib/bmf DESTDIR=$(DESTDIR) clean
+		$(MAKECMD) -C lib/bmf DESTDIR=$(DESTDIR) clean
 
 bmf_install:
-		@$(MAKECMD) -C lib/bmf DESTDIR=$(DESTDIR) install
+		$(MAKECMD) -C lib/bmf DESTDIR=$(DESTDIR) install
 
 bmf_uninstall:
-		@$(MAKECMD) -C lib/bmf DESTDIR=$(DESTDIR) uninstall
+		$(MAKECMD) -C lib/bmf DESTDIR=$(DESTDIR) uninstall
 
 dot_draw:
-		@$(MAKECMD) -C lib/dot_draw
+		$(MAKECMD) -C lib/dot_draw
 
 dot_draw_clean:
-		@$(MAKECMD) -C lib/dot_draw DESTDIR=$(DESTDIR) clean
+		$(MAKECMD) -C lib/dot_draw DESTDIR=$(DESTDIR) clean
 
 dot_draw_install:
-		@$(MAKECMD) -C lib/dot_draw DESTDIR=$(DESTDIR) install
+		$(MAKECMD) -C lib/dot_draw DESTDIR=$(DESTDIR) install
 
 dot_draw_uninstall:
-		@$(MAKECMD) -C lib/dot_draw DESTDIR=$(DESTDIR) uninstall
+		$(MAKECMD) -C lib/dot_draw DESTDIR=$(DESTDIR) uninstall
 
 dyn_gw:
-		@$(MAKECMD) -C lib/dyn_gw
+		$(MAKECMD) -C lib/dyn_gw
 
 dyn_gw_clean:
-		@$(MAKECMD) -C lib/dyn_gw DESTDIR=$(DESTDIR) clean
+		$(MAKECMD) -C lib/dyn_gw DESTDIR=$(DESTDIR) clean
 
 dyn_gw_install:
-		@$(MAKECMD) -C lib/dyn_gw DESTDIR=$(DESTDIR) install
+		$(MAKECMD) -C lib/dyn_gw DESTDIR=$(DESTDIR) install
 
 dyn_gw_uninstall:
-		@$(MAKECMD) -C lib/dyn_gw DESTDIR=$(DESTDIR) uninstall
+		$(MAKECMD) -C lib/dyn_gw DESTDIR=$(DESTDIR) uninstall
 
 dyn_gw_plain:
-		@$(MAKECMD) -C lib/dyn_gw_plain
+		$(MAKECMD) -C lib/dyn_gw_plain
 
 dyn_gw_plain_clean:
-		@$(MAKECMD) -C lib/dyn_gw_plain DESTDIR=$(DESTDIR) clean
+		$(MAKECMD) -C lib/dyn_gw_plain DESTDIR=$(DESTDIR) clean
 
 dyn_gw_plain_install:
-		@$(MAKECMD) -C lib/dyn_gw_plain DESTDIR=$(DESTDIR) install
+		$(MAKECMD) -C lib/dyn_gw_plain DESTDIR=$(DESTDIR) install
 
 dyn_gw_plain_uninstall:
-		@$(MAKECMD) -C lib/dyn_gw_plain DESTDIR=$(DESTDIR) uninstall
+		$(MAKECMD) -C lib/dyn_gw_plain DESTDIR=$(DESTDIR) uninstall
 
 httpinfo:
-		@$(MAKECMD) -C lib/httpinfo
+		$(MAKECMD) -C lib/httpinfo
 
 httpinfo_clean:
-		@$(MAKECMD) -C lib/httpinfo DESTDIR=$(DESTDIR) clean
+		$(MAKECMD) -C lib/httpinfo DESTDIR=$(DESTDIR) clean
 
 httpinfo_install:
-		@$(MAKECMD) -C lib/httpinfo DESTDIR=$(DESTDIR) install
+		$(MAKECMD) -C lib/httpinfo DESTDIR=$(DESTDIR) install
 
 httpinfo_uninstall:
-		@$(MAKECMD) -C lib/httpinfo DESTDIR=$(DESTDIR) uninstall
+		$(MAKECMD) -C lib/httpinfo DESTDIR=$(DESTDIR) uninstall
 
 jsoninfo:
-		@$(MAKECMD) -C lib/jsoninfo
+		$(MAKECMD) -C lib/jsoninfo
 
 jsoninfo_clean:
-		@$(MAKECMD) -C lib/jsoninfo DESTDIR=$(DESTDIR) clean
+		$(MAKECMD) -C lib/jsoninfo DESTDIR=$(DESTDIR) clean
 
 jsoninfo_install:
-		@$(MAKECMD) -C lib/jsoninfo DESTDIR=$(DESTDIR) install
+		$(MAKECMD) -C lib/jsoninfo DESTDIR=$(DESTDIR) install
 
 jsoninfo_uninstall:
-		@$(MAKECMD) -C lib/jsoninfo DESTDIR=$(DESTDIR) uninstall
+		$(MAKECMD) -C lib/jsoninfo DESTDIR=$(DESTDIR) uninstall
 
 mdns:
-		@$(MAKECMD) -C lib/mdns
+		$(MAKECMD) -C lib/mdns
 
 mdns_clean:
-		@$(MAKECMD) -C lib/mdns DESTDIR=$(DESTDIR) clean
+		$(MAKECMD) -C lib/mdns DESTDIR=$(DESTDIR) clean
 
 mdns_install:
-		@$(MAKECMD) -C lib/mdns DESTDIR=$(DESTDIR) install
+		$(MAKECMD) -C lib/mdns DESTDIR=$(DESTDIR) install
 
 mdns_uninstall:
-		@$(MAKECMD) -C lib/mdns DESTDIR=$(DESTDIR) uninstall
+		$(MAKECMD) -C lib/mdns DESTDIR=$(DESTDIR) uninstall
 
 #
 # no targets for mini: it's an example plugin
@@ -298,125 +298,125 @@ mdns_uninstall:
 # Android, $(REGEX_OBJS) will have all of the files needed, on all
 # other platforms, it'll be empty and therefore ignored.
 nameservice: $(REGEX_OBJS)
-		@$(MAKECMD) -C lib/nameservice clean
-		@$(MAKECMD) -C lib/nameservice
+		$(MAKECMD) -C lib/nameservice clean
+		$(MAKECMD) -C lib/nameservice
 
 nameservice_clean:
-		@$(MAKECMD) -C lib/nameservice DESTDIR=$(DESTDIR) clean
+		$(MAKECMD) -C lib/nameservice DESTDIR=$(DESTDIR) clean
 
 nameservice_install:
-		@$(MAKECMD) -C lib/nameservice DESTDIR=$(DESTDIR) install
+		$(MAKECMD) -C lib/nameservice DESTDIR=$(DESTDIR) install
 
 nameservice_uninstall:
-		@$(MAKECMD) -C lib/nameservice DESTDIR=$(DESTDIR) uninstall
+		$(MAKECMD) -C lib/nameservice DESTDIR=$(DESTDIR) uninstall
 
 p2pd:
-		@$(MAKECMD) -C lib/p2pd
+		$(MAKECMD) -C lib/p2pd
 
 p2pd_clean:
-		@$(MAKECMD) -C lib/p2pd DESTDIR=$(DESTDIR) clean
+		$(MAKECMD) -C lib/p2pd DESTDIR=$(DESTDIR) clean
 
 p2pd_install:
-		@$(MAKECMD) -C lib/p2pd DESTDIR=$(DESTDIR) install
+		$(MAKECMD) -C lib/p2pd DESTDIR=$(DESTDIR) install
 
 p2pd_uninstall:
-		@$(MAKECMD) -C lib/p2pd DESTDIR=$(DESTDIR) uninstall
+		$(MAKECMD) -C lib/p2pd DESTDIR=$(DESTDIR) uninstall
 
 pgraph:
-		@$(MAKECMD) -C lib/pgraph
+		$(MAKECMD) -C lib/pgraph
 
 pgraph_clean:
-		@$(MAKECMD) -C lib/pgraph DESTDIR=$(DESTDIR) clean
+		$(MAKECMD) -C lib/pgraph DESTDIR=$(DESTDIR) clean
 
 pgraph_install:
-		@$(MAKECMD) -C lib/pgraph DESTDIR=$(DESTDIR) install
+		$(MAKECMD) -C lib/pgraph DESTDIR=$(DESTDIR) install
 
 pgraph_uninstall:
-		@$(MAKECMD) -C lib/pgraph DESTDIR=$(DESTDIR) uninstall
+		$(MAKECMD) -C lib/pgraph DESTDIR=$(DESTDIR) uninstall
 
 pud:
-		@$(MAKECMD) -C lib/pud
+		$(MAKECMD) -C lib/pud
 
 pud_clean:
-		@$(MAKECMD) -C lib/pud DESTDIR=$(DESTDIR) clean
+		$(MAKECMD) -C lib/pud DESTDIR=$(DESTDIR) clean
 
 pud_install:
-		@$(MAKECMD) -C lib/pud DESTDIR=$(DESTDIR) install
+		$(MAKECMD) -C lib/pud DESTDIR=$(DESTDIR) install
 
 pud_uninstall:
-		@$(MAKECMD) -C lib/pud DESTDIR=$(DESTDIR) uninstall
+		$(MAKECMD) -C lib/pud DESTDIR=$(DESTDIR) uninstall
 
 quagga:
-		@$(MAKECMD) -C lib/quagga
+		$(MAKECMD) -C lib/quagga
 
 quagga_clean:
-		@$(MAKECMD) -C lib/quagga DESTDIR=$(DESTDIR) clean
+		$(MAKECMD) -C lib/quagga DESTDIR=$(DESTDIR) clean
 
 quagga_install:
-		@$(MAKECMD) -C lib/quagga DESTDIR=$(DESTDIR) install
+		$(MAKECMD) -C lib/quagga DESTDIR=$(DESTDIR) install
 
 quagga_uninstall:
-		@$(MAKECMD) -C lib/quagga DESTDIR=$(DESTDIR) uninstall
+		$(MAKECMD) -C lib/quagga DESTDIR=$(DESTDIR) uninstall
 
 secure:
-		@$(MAKECMD) -C lib/secure
+		$(MAKECMD) -C lib/secure
 
 secure_clean:
-		@$(MAKECMD) -C lib/secure DESTDIR=$(DESTDIR) clean
+		$(MAKECMD) -C lib/secure DESTDIR=$(DESTDIR) clean
 
 secure_install:
-		@$(MAKECMD) -C lib/secure DESTDIR=$(DESTDIR) install
+		$(MAKECMD) -C lib/secure DESTDIR=$(DESTDIR) install
 
 secure_uninstall:
-		@$(MAKECMD) -C lib/secure DESTDIR=$(DESTDIR) uninstall
+		$(MAKECMD) -C lib/secure DESTDIR=$(DESTDIR) uninstall
 
 sgwdynspeed:
-		@$(MAKECMD) -C lib/sgwdynspeed
+		$(MAKECMD) -C lib/sgwdynspeed
 
 sgwdynspeed_clean:
-		@$(MAKECMD) -C lib/sgwdynspeed DESTDIR=$(DESTDIR) clean
+		$(MAKECMD) -C lib/sgwdynspeed DESTDIR=$(DESTDIR) clean
 
 sgwdynspeed_install:
-		@$(MAKECMD) -C lib/sgwdynspeed DESTDIR=$(DESTDIR) install
+		$(MAKECMD) -C lib/sgwdynspeed DESTDIR=$(DESTDIR) install
 
 sgwdynspeed_uninstall:
-		@$(MAKECMD) -C lib/sgwdynspeed DESTDIR=$(DESTDIR) uninstall
+		$(MAKECMD) -C lib/sgwdynspeed DESTDIR=$(DESTDIR) uninstall
 
 tas:
-		@$(MAKECMD) -C lib/tas
+		$(MAKECMD) -C lib/tas
 
 tas_clean:
-		@$(MAKECMD) -C lib/tas DESTDIR=$(DESTDIR) clean
+		$(MAKECMD) -C lib/tas DESTDIR=$(DESTDIR) clean
 
 tas_install:
-		@$(MAKECMD) -C lib/tas DESTDIR=$(DESTDIR) install
+		$(MAKECMD) -C lib/tas DESTDIR=$(DESTDIR) install
 
 tas_uninstall:
-		@$(MAKECMD) -C lib/tas DESTDIR=$(DESTDIR) uninstall
+		$(MAKECMD) -C lib/tas DESTDIR=$(DESTDIR) uninstall
 
 txtinfo:
-		@$(MAKECMD) -C lib/txtinfo
+		$(MAKECMD) -C lib/txtinfo
 
 txtinfo_clean:
-		@$(MAKECMD) -C lib/txtinfo DESTDIR=$(DESTDIR) clean
+		$(MAKECMD) -C lib/txtinfo DESTDIR=$(DESTDIR) clean
 
 txtinfo_install:
-		@$(MAKECMD) -C lib/txtinfo DESTDIR=$(DESTDIR) install
+		$(MAKECMD) -C lib/txtinfo DESTDIR=$(DESTDIR) install
 
 txtinfo_uninstall:
-		@$(MAKECMD) -C lib/txtinfo DESTDIR=$(DESTDIR) uninstall
+		$(MAKECMD) -C lib/txtinfo DESTDIR=$(DESTDIR) uninstall
 
 watchdog:
-		@$(MAKECMD) -C lib/watchdog
+		$(MAKECMD) -C lib/watchdog
 
 watchdog_clean:
-		@$(MAKECMD) -C lib/watchdog DESTDIR=$(DESTDIR) clean
+		$(MAKECMD) -C lib/watchdog DESTDIR=$(DESTDIR) clean
 
 watchdog_install:
-		@$(MAKECMD) -C lib/watchdog DESTDIR=$(DESTDIR) install
+		$(MAKECMD) -C lib/watchdog DESTDIR=$(DESTDIR) install
 
 watchdog_uninstall:
-		@$(MAKECMD) -C lib/watchdog DESTDIR=$(DESTDIR) uninstall
+		$(MAKECMD) -C lib/watchdog DESTDIR=$(DESTDIR) uninstall
 
 
 build_all:	all switch libs
diff --git a/Makefile.inc b/Makefile.inc
index 2910fb0..160e684 100644
--- a/Makefile.inc
+++ b/Makefile.inc
@@ -218,11 +218,11 @@ quote = $(subst .,\.,$(subst /,\/,$1))
 
 # fully automatic and working dependency generation
 %.d: %.c
-	@$(filter-out $(CCACHE),$(CC)) -M $(strip $(CPPFLAGS)) "$<" | sed -e '1s/\($(call quote,$(*F))\.o\)[ :]*/$(call quote,$(*D)/\1 $@: Makefile $(TOPDIR)$(if $(TOPDIR),/)Makefile.inc) /g' >"$@"
+	$(filter-out $(CCACHE),$(CC)) -M $(strip $(CPPFLAGS)) "$<" | sed -e '1s/\($(call quote,$(*F))\.o\)[ :]*/$(call quote,$(*D)/\1 $@: Makefile $(TOPDIR)$(if $(TOPDIR),/)Makefile.inc) /g' >"$@"
 
 %.o: %.c
 	@echo "[CC] $<"
-	@$(CC) $(CFLAGS) $(CPPFLAGS) -c -o $@ $<
+	$(CC) $(CFLAGS) $(CPPFLAGS) -c -o $@ $<
 
 # we always need the includes and defines
 # for legacy since now
diff --git a/lib/arprefresh/Makefile b/lib/arprefresh/Makefile
index c860182..4665bda 100644
--- a/lib/arprefresh/Makefile
+++ b/lib/arprefresh/Makefile
@@ -48,7 +48,7 @@ default_target: $(PLUGIN_FULLNAME)
 
 $(PLUGIN_FULLNAME): $(OBJS) version-script.txt
 		@echo "[LD] $@"
-		@$(CC) $(LDFLAGS) -o $(PLUGIN_FULLNAME) $(OBJS) $(LIBS)
+		$(CC) $(LDFLAGS) -o $(PLUGIN_FULLNAME) $(OBJS) $(LIBS)
 
 install:	$(PLUGIN_FULLNAME)
 		$(STRIP) $(PLUGIN_FULLNAME)
diff --git a/lib/bmf/Makefile b/lib/bmf/Makefile
index f7fc0c8..d3c91a8 100644
--- a/lib/bmf/Makefile
+++ b/lib/bmf/Makefile
@@ -47,7 +47,7 @@ default_target: $(PLUGIN_FULLNAME)
 
 $(PLUGIN_FULLNAME): $(OBJS) version-script.txt
 		@echo "[LD] $@"
-		@$(CC) $(LDFLAGS) -o $(PLUGIN_FULLNAME) $(OBJS) $(LIBS)
+		$(CC) $(LDFLAGS) -o $(PLUGIN_FULLNAME) $(OBJS) $(LIBS)
 
 install:	$(PLUGIN_FULLNAME)
 		$(STRIP) $(PLUGIN_FULLNAME)
diff --git a/lib/dot_draw/Makefile b/lib/dot_draw/Makefile
index 3364318..e97714e 100644
--- a/lib/dot_draw/Makefile
+++ b/lib/dot_draw/Makefile
@@ -47,7 +47,7 @@ default_target: $(PLUGIN_FULLNAME)
 
 $(PLUGIN_FULLNAME): $(OBJS) version-script.txt
 		@echo "[LD] $@"
-		@$(CC) $(LDFLAGS) -o $(PLUGIN_FULLNAME) $(OBJS) $(LIBS)
+		$(CC) $(LDFLAGS) -o $(PLUGIN_FULLNAME) $(OBJS) $(LIBS)
 
 install:	$(PLUGIN_FULLNAME)
 		$(STRIP) $(PLUGIN_FULLNAME)
diff --git a/lib/dyn_gw/Makefile b/lib/dyn_gw/Makefile
index 9246c42..92136da 100644
--- a/lib/dyn_gw/Makefile
+++ b/lib/dyn_gw/Makefile
@@ -49,7 +49,7 @@ default_target: $(PLUGIN_FULLNAME)
 
 $(PLUGIN_FULLNAME): $(OBJS) version-script.txt
 		@echo "[LD] $@"
-		@$(CC) $(LDFLAGS) -o $(PLUGIN_FULLNAME) $(OBJS) $(LIBS)
+		$(CC) $(LDFLAGS) -o $(PLUGIN_FULLNAME) $(OBJS) $(LIBS)
 
 install:	$(PLUGIN_FULLNAME)
 		$(STRIP) $(PLUGIN_FULLNAME)
diff --git a/lib/dyn_gw_plain/Makefile b/lib/dyn_gw_plain/Makefile
index f1cbe5e..05acf61 100644
--- a/lib/dyn_gw_plain/Makefile
+++ b/lib/dyn_gw_plain/Makefile
@@ -47,7 +47,7 @@ default_target: $(PLUGIN_FULLNAME)
 
 $(PLUGIN_FULLNAME): $(OBJS) version-script.txt
 		@echo "[LD] $@"
-		@$(CC) $(LDFLAGS) -o $(PLUGIN_FULLNAME) $(OBJS) $(LIBS)
+		$(CC) $(LDFLAGS) -o $(PLUGIN_FULLNAME) $(OBJS) $(LIBS)
 
 install:	$(PLUGIN_FULLNAME)
 		$(STRIP) $(PLUGIN_FULLNAME)
diff --git a/lib/httpinfo/Makefile b/lib/httpinfo/Makefile
index 7e388d6..cdc06c3 100644
--- a/lib/httpinfo/Makefile
+++ b/lib/httpinfo/Makefile
@@ -56,7 +56,7 @@ endif
 
 $(PLUGIN_FULLNAME): $(OBJS) version-script.txt
 		@echo "[LD] $@"
-		@$(CC) $(LDFLAGS) -o $(PLUGIN_FULLNAME) $(OBJS) $(LIBS)
+		$(CC) $(LDFLAGS) -o $(PLUGIN_FULLNAME) $(OBJS) $(LIBS)
 
 install:	$(PLUGIN_FULLNAME)
 		$(STRIP) $(PLUGIN_FULLNAME)
diff --git a/lib/jsoninfo/Makefile b/lib/jsoninfo/Makefile
index 2c714a4..d55da6b 100644
--- a/lib/jsoninfo/Makefile
+++ b/lib/jsoninfo/Makefile
@@ -47,7 +47,7 @@ default_target: $(PLUGIN_FULLNAME)
 
 $(PLUGIN_FULLNAME): $(OBJS) version-script.txt
 		@echo "[LD] $@"
-		@$(CC) $(LDFLAGS) -o $(PLUGIN_FULLNAME) $(OBJS) $(LIBS)
+		$(CC) $(LDFLAGS) -o $(PLUGIN_FULLNAME) $(OBJS) $(LIBS)
 
 install:	$(PLUGIN_FULLNAME)
 		$(STRIP) $(PLUGIN_FULLNAME)
diff --git a/lib/mdns/Makefile b/lib/mdns/Makefile
index e2f71b8..fcc9e5f 100644
--- a/lib/mdns/Makefile
+++ b/lib/mdns/Makefile
@@ -22,7 +22,7 @@ default_target: $(PLUGIN_FULLNAME)
 
 $(PLUGIN_FULLNAME): $(OBJS) version-script.txt
 		@echo "[LD] $@"
-		@$(CC) $(LDFLAGS) -o $(PLUGIN_FULLNAME) $(OBJS) $(LIBS)
+		$(CC) $(LDFLAGS) -o $(PLUGIN_FULLNAME) $(OBJS) $(LIBS)
 
 install:	$(PLUGIN_FULLNAME)
 		$(STRIP) $(PLUGIN_FULLNAME)
diff --git a/lib/mini/Makefile b/lib/mini/Makefile
index 4bf26dd..ae55f40 100644
--- a/lib/mini/Makefile
+++ b/lib/mini/Makefile
@@ -47,7 +47,7 @@ default_target: $(PLUGIN_FULLNAME)
 
 $(PLUGIN_FULLNAME): $(OBJS) version-script.txt
 		@echo "[LD] $@"
-		@$(CC) $(LDFLAGS) -o $(PLUGIN_FULLNAME) $(OBJS) $(LIBS)
+		$(CC) $(LDFLAGS) -o $(PLUGIN_FULLNAME) $(OBJS) $(LIBS)
 
 install:	$(PLUGIN_FULLNAME)
 		$(STRIP) $(PLUGIN_FULLNAME)
diff --git a/lib/nameservice/Makefile b/lib/nameservice/Makefile
index 9ba270a..2dbbe73 100644
--- a/lib/nameservice/Makefile
+++ b/lib/nameservice/Makefile
@@ -60,7 +60,7 @@ default_target: $(PLUGIN_FULLNAME)
 
 $(PLUGIN_FULLNAME): $(OBJS) version-script.txt
 		@echo "[LD] $@"
-		@$(CC) $(LDFLAGS) -o $(PLUGIN_FULLNAME) $(OBJS) $(LIBS)
+		$(CC) $(LDFLAGS) -o $(PLUGIN_FULLNAME) $(OBJS) $(LIBS)
 
 install:	$(PLUGIN_FULLNAME)
 		$(STRIP) $(PLUGIN_FULLNAME)
diff --git a/lib/p2pd/Makefile b/lib/p2pd/Makefile
index c61d560..48f0d2c 100644
--- a/lib/p2pd/Makefile
+++ b/lib/p2pd/Makefile
@@ -22,7 +22,7 @@ default_target: $(PLUGIN_FULLNAME)
 
 $(PLUGIN_FULLNAME): $(OBJS) version-script.txt
 		@echo "[LD] $@"
-		@$(CC) $(LDFLAGS) -o $(PLUGIN_FULLNAME) $(OBJS) $(LIBS)
+		$(CC) $(LDFLAGS) -o $(PLUGIN_FULLNAME) $(OBJS) $(LIBS)
 
 install:	$(PLUGIN_FULLNAME)
 		$(STRIP) $(PLUGIN_FULLNAME)
diff --git a/lib/pgraph/Makefile b/lib/pgraph/Makefile
index af42ab1..763df00 100644
--- a/lib/pgraph/Makefile
+++ b/lib/pgraph/Makefile
@@ -47,7 +47,7 @@ default_target: $(PLUGIN_FULLNAME)
 
 $(PLUGIN_FULLNAME): $(OBJS) version-script.txt
 		@echo "[LD] $@"
-		@$(CC) $(LDFLAGS) -o $(PLUGIN_FULLNAME) $(OBJS) $(LIBS)
+		$(CC) $(LDFLAGS) -o $(PLUGIN_FULLNAME) $(OBJS) $(LIBS)
 
 install:	$(PLUGIN_FULLNAME)
 		$(STRIP) $(PLUGIN_FULLNAME)
diff --git a/lib/quagga/Makefile b/lib/quagga/Makefile
index b97aef8..ba977fc 100644
--- a/lib/quagga/Makefile
+++ b/lib/quagga/Makefile
@@ -57,7 +57,7 @@ default_target: $(PLUGIN_FULLNAME)
 
 $(PLUGIN_FULLNAME): $(OBJS) version-script.txt
 		@echo "[LD] $@"
-		@$(CC) $(LDFLAGS) -o $(PLUGIN_FULLNAME) $(OBJS) $(LIBS)
+		$(CC) $(LDFLAGS) -o $(PLUGIN_FULLNAME) $(OBJS) $(LIBS)
 
 install:	$(PLUGIN_FULLNAME)
 		$(STRIP) $(PLUGIN_FULLNAME)
diff --git a/lib/secure/Makefile b/lib/secure/Makefile
index adb5e1e..df45230 100644
--- a/lib/secure/Makefile
+++ b/lib/secure/Makefile
@@ -52,7 +52,7 @@ default_target: $(PLUGIN_FULLNAME)
 
 $(PLUGIN_FULLNAME): $(OBJS) version-script.txt
 		@echo "[LD] $@"
-		@$(CC) $(LDFLAGS) -o $(PLUGIN_FULLNAME) $(OBJS) $(LIBS)
+		$(CC) $(LDFLAGS) -o $(PLUGIN_FULLNAME) $(OBJS) $(LIBS)
 
 install:	$(PLUGIN_FULLNAME)
 		$(STRIP) $(PLUGIN_FULLNAME)
diff --git a/lib/sgwdynspeed/Makefile b/lib/sgwdynspeed/Makefile
index d012702..79afbe4 100644
--- a/lib/sgwdynspeed/Makefile
+++ b/lib/sgwdynspeed/Makefile
@@ -30,7 +30,7 @@ default_target: $(PLUGIN_FULLNAME)
 
 $(PLUGIN_FULLNAME): $(OBJS) version-script.txt
 	@echo "[LD] $@"
-	@$(CC) $(LDFLAGS) -o $(PLUGIN_FULLNAME) $(OBJS) $(LIBS)
+	$(CC) $(LDFLAGS) -o $(PLUGIN_FULLNAME) $(OBJS) $(LIBS)
 
 install: all
 	$(INSTALL_LIB)
@@ -46,12 +46,12 @@ uninstall:
 clean:
 	@echo "[$@]"
 	@rm -f $(OBJS) $(SRCS:%.c=%.d) "$(PLUGIN_FULLNAME)"
-	@$(MAKE) -C doc clean
+	$(MAKE) -C doc clean
 
 doc:
-	@$(MAKE) -C doc all
+	$(MAKE) -C doc all
 
 doc-clean:
-	@$(MAKE) -C doc clean
+	$(MAKE) -C doc clean
 
 endif
diff --git a/lib/tas/Makefile b/lib/tas/Makefile
index c164dcf..056353b 100644
--- a/lib/tas/Makefile
+++ b/lib/tas/Makefile
@@ -61,7 +61,7 @@ default_target: $(PLUGIN_FULLNAME)
 
 $(PLUGIN_FULLNAME): $(OBJS) version-script.txt
 		@echo "[LD] $@"
-		@$(CC) $(LDFLAGS) -o $(PLUGIN_FULLNAME) $(OBJS) $(LIBS)
+		$(CC) $(LDFLAGS) -o $(PLUGIN_FULLNAME) $(OBJS) $(LIBS)
 
 # this is not nice but there is no nicer solution
 src/lua/lapi.o src/lua/ltable.o:  CFLAGS += -Wno-cast-qual
diff --git a/lib/txtinfo/Makefile b/lib/txtinfo/Makefile
index 63c2920..e708584 100644
--- a/lib/txtinfo/Makefile
+++ b/lib/txtinfo/Makefile
@@ -47,7 +47,7 @@ default_target: $(PLUGIN_FULLNAME)
 
 $(PLUGIN_FULLNAME): $(OBJS) version-script.txt
 		@echo "[LD] $@"
-		@$(CC) $(LDFLAGS) -o $(PLUGIN_FULLNAME) $(OBJS) $(LIBS)
+		$(CC) $(LDFLAGS) -o $(PLUGIN_FULLNAME) $(OBJS) $(LIBS)
 
 install:	$(PLUGIN_FULLNAME)
 		$(STRIP) $(PLUGIN_FULLNAME)
diff --git a/lib/watchdog/Makefile b/lib/watchdog/Makefile
index b22c094..0fa1ceb 100644
--- a/lib/watchdog/Makefile
+++ b/lib/watchdog/Makefile
@@ -47,7 +47,7 @@ default_target: $(PLUGIN_FULLNAME)
 
 $(PLUGIN_FULLNAME): $(OBJS) version-script.txt
 		@echo "[LD] $@"
-		@$(CC) $(LDFLAGS) -o $(PLUGIN_FULLNAME) $(OBJS) $(LIBS)
+		$(CC) $(LDFLAGS) -o $(PLUGIN_FULLNAME) $(OBJS) $(LIBS)
 
 install:	$(PLUGIN_FULLNAME)
 		$(STRIP) $(PLUGIN_FULLNAME)
